x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_flights/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "sample_dataset_flights",
```

Comment:
ID of the module should match the directory name. `sample_data_flights` perhaps?